chat-plugins/info.js

```diff
@@ -460,6 +288,15 @@ var commands = exports.commands = {
 		}
 		this.sendReply(buffer);
 	},
+	datahelp:
+		"/data [pokemon/item/move/ability] - Get details on this pokemon/item/move/ability/nature." + "\n" +
```

Comment:
I think I prefer it the other way, unless someone else has stronger preferences? @Slayer95? @Joimer?